assets/src/scss/layout/navbar/_search.scss

```diff
@@ -58,7 +58,7 @@
     top: $menu-height;
   }
 
-  @include x-large-and-up {
+  @include bs-xx-large-and-up {
     background: var(--top-navigation--background, inherit);
```

Comment:
Are we replacing the XL breakpoint or the XXL?

Reply:
Yes but only here. The main reason is because the Design team want to use 1400px ([BS's XXL breakpoint](https://getbootstrap.com/docs/5.0/layout/breakpoints/#available-breakpoints)) instead of 1200px.